app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<section>
+  <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+    <%= render "devise/shared/error_messages", resource: resource %>
+
+    <nav class="nav navbar-light bg-primary">
+      <div class="container py-1 px-3 d-flex justify-content-between align-items-center">
+        <%= link_to fa_icon('arrow-left'), unauthenticated_root_path, class: "fs-6 fw-normal text-white"  %>
+        <h1 class="fs-6 fw-normal text-white pt-2">REGISTER</h1>
+        <%= f.submit "Next", class: "text-white fs-6 bg-transparent border-0 fw-normal" %>
+      </div>
+    </nav>
```

Comment:
- [ ] Kindly change the background color of the header from all screen to match the design guidline

| Your design  | Design guidline |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/53402600/227621332-bf5aadd7-564b-46cd-af36-0708b8c7adb7.png) | ![image](https://user-images.githubusercontent.com/53402600/227621411-77c553d2-7c50-458a-9e4a-735123c3fbc9.png) |